setup.py

```diff
@@ -244,6 +244,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 ]
 databricks = [
     'requests>=2.26.0, <3',
+    'aiohttp>=3.6.3, <4',
+    'asynctest~=0.13',
```

Comment:
This should not be a part of the provider extra, but in the `tests` extra instead.